gtk/transmission-gtk.appdata.xml.in

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2014 Richard Hughes <richard@hughsie.com>
+Copyright 2017 Endless Mobile, Inc.
```

Comment:
Was your time spent on the PR covered by this company? If not, might be a better idea to provide your own name here. Just double-checking.